packages/build/src/plugins_core/functions/utils.js

```diff
@@ -0,0 +1,70 @@
+'use strict'
```

Comment:
This is a new file with functions extracted from `packages/build/src/plugins_core/functions/index.js`.